src/BloomBrowserUI/lib/localizationManager/localizationManager.ts

```diff
@@ -160,6 +160,29 @@ class LocalizationManager {
 
     */
     asyncGetTextInLang(id: string, englishText: string, langId: string, ...args): JQueryPromise {
+        return this.asyncGetTextInLangCommon(id, englishText, langId, args)
+    }
+
+    /* Returns a promise to get the translation in the current UI language
+     *
+     * @param {String[]} args (optional): can be used as parameters to insert into c#-style parameterized strings
+     *  @example
+     * asyncGetText('topics.health','Health')
+     *      .done(translation => {
+     *          $(this).text(translation);
+     *      })
+     *      .fail($(this).text("?Health?"));
+     * @example
+     * asyncGetTextInLang('topics.health','My name is {0}", "John")
+     *      .done(translation => {
+     *          $(this).text(translation);
+     *      });
+    */
+    asyncGetText(id: string, englishText: string, ...args): JQueryPromise {
+        return this.asyncGetTextInLangCommon(id, englishText, "UI", args);
+    }
+
+    asyncGetTextInLangCommon(id: string, englishText: string, langId: string, args): JQueryPromise {
```

Comment:
I'm not sure I'm understanding. It appears that asyncGetTextInLang has the same arguments and behavior as asyncGetTextInLangCommon, except that one has ...args while one has simply args. The ...args converts all remaining arguments into an array which both methods pass to the common one. so if asyncGetText called asyncGetTextInLang, instead of passing an array of two strings, it would receive an array of one element, an array of two strings. One way around it would be for asyncGetText to return asyncGetTextInLang.apply(this, [id, englishText, "UI"].concat(args)). But I'm not sure this is any better than what you've done.